app/assets/sass/application.scss

```diff
@@ -4,6 +4,7 @@
 // $govuk-compatibility-govukfrontendtoolkit: true;
 
 $govuk-global-styles: true;
+$moj-images-path: "../../../node_modules/@ministryofjustice/frontend/moj/assets/images/";
```

Comment:
All these hardwired references to `node_modules` will want addressing at some point, it's not the usual way to reference this and should they change the names or folder structure, which they could do as they don't say this is the way to reference the files, then it could all break.

However, this is as it currently is so it's not going to stop me approving 🙂 Possibly one for @simonnebMOJ to add a ticket for separately?

Reply:
@psoleckimoj Ah yes, the app kept crashing when it wasn't referenced like this so yes def one to have a look at making a ticket for @simonnebMOJ 

Same with all the moj component imports a few lines down in this file. For some reason I couldn't import them with just  `@import'../../../node_modules/@ministryofjustice/frontend/moj/components/all `